src/WebJobs.Script/Utility.cs

```diff
@@ -673,9 +673,15 @@ internal static bool IsFunctionMetadataLanguageSupportedByWorkerRuntime(Function
             {
                 return true;
             }
+
             return !string.IsNullOrEmpty(functionMetadata.Language) && functionMetadata.Language.Equals(workerRuntime, StringComparison.OrdinalIgnoreCase);
         }
 
+        internal static bool IsFunctionMetadataLanguageSupportedByWorkerRuntime(FunctionMetadata functionMetadata, IList<RpcWorkerConfig> workerConfig)
```

Comment:
nit -  for consistency with CanWorkerIndex method params, could change param name to `workerConfigs`?